### PR TITLE
Repocheck

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by kbaikov** [comment]
+1

**Comment by toabctl** [comment]
You could add ```- openSUSE_Leap_42.2``` but can also be done in another PR. +1

File: scripts/jenkins/jobs-obs/openstack-repocheck.yaml [hunk 1]
```diff
@@ -46,11 +46,11 @@
...
          ["Cloud:OpenStack:Factory"].contains(project) && ["SLE_11_SP3", "SLE_12", "SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1"].contains(repository) ||
-         ["Cloud:OpenStack:Icehouse", "Cloud:OpenStack:Icehouse:Staging"].contains(project) && ["SLE_12", "SLE_12_SP1", "SLE_12_SP2", "openSUSE_13.2", "openSUSE_Leap_42.1"].contains(repository) ||
          ["Cloud:OpenStack:Juno", "Cloud:OpenStack:Juno:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1"].contains(repository)  ||
          ["Cloud:OpenStack:Kilo", "Cloud:OpenStack:Kilo:Staging"].contains(project) && ["SLE_12_SP1", "SLE_12_SP2", "openSUSE_Leap_42.1", "SLE_11_SP3"].contains(repository)  ||
          ["Cloud:OpenStack:Liberty", "Cloud:OpenStack:Liberty:Staging"].contains(project) && ["SLE_11_SP3", "SLE_12_SP2"].contains(repository) ||
          ["Cloud:OpenStack:Mitaka", "Cloud:OpenStack:Mitaka:Staging"].contains(project) && ["openSUSE_13.2", "SLE_12", "SLE_11_SP3"].contains(repository)
+         ["Cloud:OpenStack:Newton", "Cloud:OpenStack:Newton:Staging"].contains(project) && ["openSUSE_13.2", "SLE_12", "SLE_12_SP1", "SLE_11_SP3", "openSUSE_Leap_42.1"].contains(repository)
```
Comment (bmwiedemann) [inline scripts/jenkins/jobs-obs/openstack-repocheck.yaml]: -0.5 SLE_11_SP3 is long EOL and 13.2 will soon be

Comment (toabctl) [inline scripts/jenkins/jobs-obs/openstack-repocheck.yaml]: @bmwiedemann It's the blacklist

**Pull request merged**